pkg/scheduler/plugins/gang/gang.go

```diff
@@ -75,10 +75,18 @@ func (gp *gangPlugin) OnSessionOpen(ssn *framework.Session) {
 	preemptableFn := func(preemptor *api.TaskInfo, preemptees []*api.TaskInfo) []*api.TaskInfo {
 		var victims []*api.TaskInfo
 
+		// If job is already ready, no need to preempt others in Gang plgin
+		preemptorJob := ssn.Jobs[preemptor.Job]
+		readyNum := preemptorJob.ReadyTaskNum()
+		ready := readyNum >= preemptorJob.MinAvailable || preemptorJob.MinAvailable == 1
```

Comment:
Why if `preemptorJob.MinAvailable == 1`, do not preempt ?

Reply:
If preemptorJob.MinAvaiable ==1, it is the scope of drf preemption. We use the drf share to preempt.